Resources/public/css/views/content-creation.css

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+.ez-view-contentcreationwidgetview {
+    width: 25rem;
+    height: 42rem;
+    padding: 2rem;
+    box-sizing: border-box;
+}
+
+.ez-view-contentcreationwidgetview .on-page-two {
+    transform: translateX(-100%);
+}
+
+.ez-view-contentcreationwidgetview .cof-content-creation__header__headline {
+    margin: 0;
+}
+
+.ez-view-contentcreationwidgetview .cof-content-creation__content {
+    display: flex;
+    height: 34rem;
+    overflow: hidden;
+}
+
+.ez-view-contentcreationwidgetview .cof-content-creation__page-one,
+.ez-view-contentcreationwidgetview .cof-content-creation__page-two {
+    flex: 1 0 100%;
```

Comment:
Why did you decide to use `flex: 1 0 100%` here?
Is it important to have flexbox here?